### DISCUSSION TITLE
Allow deletion of files from git lfs

### DISCUSSION THREAD

**Discussion by BrentMifsud** [community/community#4692] [Other Feature Feedback, Questions, & Ideas]
Currently the only option to clear git lfs is to delete your repo.

This is not a suitable option. You lose all your tickets, all your stars, etc...

There really needs to be a non-nuclear option for cleaning up old unused git lfs files from the remote.

**Comment by nodrock** [comment]
For us it would be sufficient to have button to delete all LFS objects from repo somewhere in danger zone of repo settings.

**Reply by BrentMifsud** [reply]
After some digging we discovered you could contact GitHub support to get it done.

But I still think there should be a button to clear git LFS. Should require admin privileges as it can be just as destructive as deleting a repo.

**Reply by nodrock** [reply]
Yeah that's how I actually made it. But I think the same it should be in repo settings.

**Comment by Thaina** [comment]
I also wish that we should have common management command. Such as delete files that has no dependency on the current head. Or delete every file that have no dependencies in the history

But ability to clear everything in lfs and reupload without destroy the repo itself is a very first priority. If it was a very heavy process or exploitable I still accept that it could only press once a month or could only queue it and it would finish at later date, or anything similar
